### DISCUSSION TITLE
how to delete a row from a table

### DISCUSSION THREAD

**Discussion by andrewchenshx** [TriliumNext/Trilium#3175] [General]
say there is a table with 10 rows.
i can insert rows or columns with the pop-up toolbar, while I don't know how to delete some rows.
thanks

**Comment by XXXJetfireXXX** [comment]
1) Select rows you want to delete
2) In "row" menu choose "delete row"
![image](https://user-images.githubusercontent.com/43646178/192960799-d02dc23e-f170-490d-b33c-c0ca2f570068.png)
